### ISSUE TITLE
Thanks. Do you know of a linux solution?

### ISSUE THREAD

**Issue by GreenDude120** [rnyit/pvpgn-password-request#1]
I host pvpgn on linux so I cant use this method. But I'd like to be able to do email recovery. Heard of any solution?

**Comment by rnyweb** [comment]
Maybe too late reply. There is no solution for pvpgn to recover the password other than this. Did u try it? It should work 99%. If is not working I will need some feedback, logs and so on and I will make it for linux as well.

**Comment by rnyit** [comment]
Maybe too late for my reply. There is no solution for pvpgn to recover the password other than this. Did u try it? It should work 99%. If is not working I will need some feedback, logs and so on and I will make it for linux as well.

**Comment by GreenDude120** [comment]
i cant use it because I run pvpgn on Linux OS. and your solution is for windows OS. i was wondering if you would make it for linux too

**Comment by rnyit** [comment]
Well, if pvpgn does run the same as windows and it does output the same logs format it should work. This script is just tracking the log file and when a request was made via pvpgn the script will lookup for the email stored for the user that requested the password to be reseted and it will send an email. So, try to compare the logs that are generated from linux and windows and if they look the same, it will work 100%.

**Comment by rnyit** [comment]
The only thing that **needs** to be modified are the slashes for the path of directories.

**Comment by GreenDude120** [comment]
Yes the log is the same but your files are for windows. are you planning on doing linux too ? also feel free to add me on discord GreenDude#0001

**Comment by rnyit** [comment]
Well I will, why not. I don't use social media, I will create a discord now.
